### ISSUE TITLE
Compiling UFO failed: module 'uharfbuzz' has no attribute 'RepackerError'

### ISSUE THREAD

**Issue by eduairet** [googlefonts/fontmake#884]
Hi, I'm using the `fontmake -m MyDesignSpace.designspace --check-compatibility` command and I'm getting the message:

`Compiling UFO failed: module 'uharfbuzz' has no attribute 'RepackerError'`

I think it's connected to other error I'm also getting when trying to make instances with python

`In '../../../../03-sources/Roman/MyDesignSpace.designspace': Generating fonts from Designspace failed: <method 'load_module' of 'zipimport.zipimporter' objects> returned NULL without setting an error`

Does someone know what's happening?

**Comment by vv-monsalve** [comment]
Hello, 

I'm receiving the same Error when trying to produce the Helius font (WIP project) with Builder when using `Fontmake 3.3.0`. Command used `gftools builder config.yml`

I've inspected the design.space file and everything seems ok in it, but the same error is still present when running the command `fontmake -m ./master_ufo/Helius.designspace -o variable`.

Link to the source file: https://github.com/vv-monsalve/Helius-Font/tree/qaHelius

**Comment by chrissimpkins** [comment]
Does it work if you add `uharfbuzz==0.26.0` as a build dependency in your requirements.txt file and re-install all build dependencies in your build environment?

It looks like the `RepackerError` was added to uharfbuzz back in March and released in v0.23.0

https://github.com/harfbuzz/uharfbuzz/pull/113

**Comment by aminabedi68** [comment]
same error with `fontmake 3.4.0a2` and a v5.0 designspace:
```
fontmake: Error: In 'Mikhak(despv5).designspace': 
Generating fonts from Designspace failed: module 'uharfbuzz' has no attribute 'RepackerError'
```
it's an opensource project, if it helps i can share the sources.

**Comment by eduairet** [comment]
> 

This worked for me 🙃

**Comment by vv-monsalve** [comment]
Hi @chrissimpkins 

Adding `uharfbuzz==0.26.0` didn't work on my end :(

Still receiving:

`fontmake.errors.FontmakeError: In 'Helius.glyphs' -> 'master_ufo/Helius.designspace': Generating fonts from Designspace failed: module 'uharfbuzz' has no attribute 'RepackerError'`

Compiling is working ok under another venv with `gftools 0.9.10` though.

@m4rc1e what could be the situation? - [Source file](https://github.com/vv-monsalve/Helius-Font/blob/qaHelius/Sources/Helius.glyphs) in qaHelius branch

**Comment by m4rc1e** [comment]
It worked for me using a fresh venv and the latest [gftools](https://github.com/googlefonts/gftools/releases/tag/v0.9.12)

**Comment by vv-monsalve** [comment]
Ok. I'll give it a try under the newly released `gftools v0.9.12`.
It didn't work yesterday under multiple new venvs using `0.9.11`.

**Comment by vv-monsalve** [comment]
It worked under `v0.9.12`. Thanks @m4rc1e
